src/activity-pub-db.js

```diff
@@ -221,7 +222,7 @@ async function firstTimeSetup(actorName) {
 
 function setup() {
   // activitypub not set up yet, skip until we have the data we need
-  if (actorInfo.disabled) {
+  if (false) {
```

Comment:
I have temporarily removed the disabled flag across this PR. Still thinking about what this should be—maybe some UI onboarding on first run, or we disable AP until it's enabled in admin settings?

Reply:
I think I'd like a non-AP mode to be the default for at least a little longer because it's useful to not get in your way when you're running on localhost to work on some non-federated part of the app. I think it's fine to kick that into the admin settings, update the instructions to send people there as quickly as possible, and _possibly_ add some pointers in the future like a localstorage-dismissable banner at the top of the screen that tells you you're in non-AP mode when you're logged in as admin? 🤔

Reply:
@ckolderup what you are describing is very similar to the Owncast configuration. Get all the bits and bobbles setup first, then you can flip on AP mode if you want.

https://owncast.online/docs/social/

![image](https://github.com/ckolderup/postmarks/assets/6627582/3c87bdc6-e30f-4435-82ae-0aa3849c86c1)

Reply:
@jeffsikes ooh, interesting, thank you! This is a great comparison.